src/components/access_group/AccessGroupEdit.vue

```diff
@@ -135,8 +124,65 @@ export default {
         component: 'AccessGroupHistoryEdit',
       });
     }
+
+    this.loading = false;
   },
   methods: {
+    async fetchAccessGroupInformation() {
+      try {
+        const groupData = await accessGroupApi.execute({
+          path: 'group/get',
+          endpointArguments: [this.groupname],
+        });
+        const groupInformation = {};
+        groupInformation.group = new GroupViewModel(groupData.group);
+        groupInformation.membership = new MembershipModel(groupData.membership);
+        groupInformation.memberCount = !groupData.member_count
+          ? 0
+          : groupData.member_count;
+        groupInformation.invitationCount = !groupData.invitation_count
+          ? 0
+          : groupInformation.invitation_count;
+        groupInformation.renewalCount = !groupData.renewal_count
+          ? 0
+          : groupInformation.renewal_count;
+        groupInformation.requestCount = groupData.request_count;
+        groupInformation.invitationConfig = groupData.invitation;
+        groupInformation.isCurator = Boolean(groupData.curator);
+        groupInformation.isMember = Boolean(groupData.member);
+        this.groupInformation = groupInformation;
+      } catch (e) {
+        console.log(e.message);
+        throw new Error(e.message);
+      }
+    },
+    async fetchTOS() {
+      try {
+        this.tos =
+          (await accessGroupApi.execute({
+            path: 'terms/get',
+            endpointArguments: [this.groupname],
+          })) || '';
+      } catch (e) {
+        console.log(e.message);
+        throw new Error(e.message);
+      }
+    },
+    async fetchMembers() {
+      try {
+        const memberData = await accessGroupApi.execute({
+          path: 'members/get',
+          endpointArguments: [this.groupname, { sort: 'role-asc' }],
+        });
+        const members = memberData.members.map(
+          (member) => new DisplayMemberViewModel(member),
+        );
+        this.memberList = members;
+      } catch (e) {
+        console.log(e.message);
```

Comment:
Looks like this try catch block is everywhere. Moving the `console.log` to the `execute` method allows us to remove those (and maybe use `console.error`)